yb-voyager/cmd/importSchemaYugabyteDB.go

```diff
@@ -99,6 +99,14 @@ func YugabyteDBImportSchema(target *tgtdb.Target, exportDir string) {
 							os.Exit(1)
 						}
 					}
+				} else if strings.Contains(err.Error(), "multiple primary keys") {
```

Comment:
Yeah, I agree with @amit-yb this might be a genuine error in some corner case maybe user adds an extra ALTER command to create one more PK. 
A better and strict check would have been to check for string + make system catalog query to find if already a same-name/same-column index exists on this table.